contributors/design-proposals/raw-block-pv.md

```diff
@@ -0,0 +1,571 @@
+# Local Raw Block Consumption via Persistent Volume Source
```

Comment:
We can remove "local" from the title since both local and network attached are target of this proposal.
Can we use same terminology throughout the spec?
- block volume
- raw block volume
- raw block device